<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ALU</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 185, 228);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPlainTextEdit" name="entradaHexA">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(193, 232, 240);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>resultado A en hexadecimal</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="entradaHexB">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(193, 232, 240);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>resultado B en hexadecimal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entradaA">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#entradaA:hover{
	
background-color: rgb(160, 240, 236);
}

#entradaA{
background-color: rgb(193, 232, 240);
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>inserte el numero A</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entradaB">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#entradaB:hover{
	background-color: rgb(160, 240, 236);
}

#entradaB{
background-color: rgb(193, 232, 240);
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>inserte el numero B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dividir">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#transformar{
	
	
	background-color: rgb(120, 227, 197);
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 5px;
}

#transformar:pressed{
	
	background-color: rgb(49, 151, 97);
}

#dividir{
	
	
	background-color: rgb(147, 179, 238);
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 5px;
}

#dividir:pressed{
	
	background-color: rgb(52, 101, 164);
}</string>
    </property>
    <property name="text">
     <string>DIVISION</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">grupo</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="multiplicar">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#transformar{
	
	
	background-color: rgb(120, 227, 197);
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 5px;
}

#transformar:pressed{
	
	background-color: rgb(49, 151, 97);
}
#multiplicar{
	
	
	background-color: rgb(147, 179, 238);
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 5px;
}

#multiplicar:pressed{
	
	
	background-color: rgb(52, 101, 164);
}</string>
    </property>
    <property name="text">
     <string>MULTIPLICACION</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">grupo</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="sumar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#transformar{
	
	
	background-color: rgb(120, 227, 197);
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 5px;
}

#transformar:pressed{
	
	background-color: rgb(49, 151, 97);
}
#sumar{
		
	background-color: rgb(147, 179, 238);
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 5px;
}

#sumar:pressed{
	
	background-color: rgb(52, 101, 164);
}</string>
    </property>
    <property name="text">
     <string>SUMA</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">grupo</string>
    </attribute>
   </widget>
   <widget class="QPlainTextEdit" name="salida">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#salida:hover{
background-color: rgb(160, 240, 236);
}
#salida{
background-color: rgb(193, 232, 240);
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Resultado operacion</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="salidaHexadecimal">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(193, 232, 240);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>resultado operacion en hexadecimal</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuIEEE754_CALCULATOR">
    <property name="title">
     <string>IEEE754 CALCULATOR 30000</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuIEEE754_CALCULATOR"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="grupo"/>
 </buttongroups>
</ui>
